client/index.html

```diff
@@ -1,11 +1,473 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <template data-type="pug" data-src="./view/base/meta.pug" />
-    <link rel="stylesheet" href="./index.css">
-    <script type="module" src="./index.js" defer></script>
-  </head>
-  <body>
-    <template data-type="pug" data-src="./view/browsers/browsers.pug" />
-  </body>
+<head>
+  <link rel="stylesheet" href="./index.css">
+  <script type="module" src="./index.js" defer></script>
+  <meta name="color-scheme" content="dark light">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+  <article class="Article Article--intro">
+      <div class="Article--intro__header">
+        <h1 class="Article__h1">
+          <img src="./assets/browserlist_logo.svg" role="presentation">
+          Browserslist
+        </h1>
+        <a class="Badge" href="https://github.com/browserslist/browserslist">
+          <img src="./assets/github_logo.svg" role="presentation">
```

Comment:
We still have a11y issue. The problem was not lack of `alt`. The problem was that blind user will not have explanation what is this number about.

I think the best way is `alt` but we can also use some `aria-` tags.

Reply:
What do you think about this solution?
<img width="702" alt="image" src="https://user-images.githubusercontent.com/27442574/182603949-c9d67192-cea4-440c-825d-05d9dc8a4a5c.png">
<img width="275" alt="image" src="https://user-images.githubusercontent.com/27442574/182603971-88751178-8306-4e50-953a-293dad867dc6.png">

Reply:
Works, but looks more complicated rather just using `alt` for images.

Reply:
If i add a alt, it will be less undersandable, i guess... If you agree with my way, i am doing it)

Reply:
Nope, this time let’s use the simplest way. We already have a big DOM tree. Increasing it without a good rational reason will not be a good idea.

Also, I do not want to spend time of reviewing the best way to hide text for screen readers. There is a small holy war about techniques, some say that [`visibility: hidden` is bad](https://webaim.org/techniques/css/invisiblecontent/) since screen reader check this CSS property and will ignore text since it is invisible.

> isplay:none or visibility: hidden
> These styles will hide content from all users. The content is removed from the visual flow of the page and is ignored by screen readers. Do not use this CSS if you want the content to be read by a screen reader. But DO use it for content you want hidden from all users.

Reply:
It is still not a recommended “screen reader only” technique.

Let’s move to alt.

Good a11y is not about crazy hacks, but about simple predicted solutions.

Also, adding role=presentation is not good semantic. These images are critical to understandard information. For instance, if browser can’t load image with role=presentation if will hide it. But if image contains alt, browser will show alt text if image can’t be loaded (it is what we need here).